packages/itwin/ec3-widget/src/components/EC3/EC3AuthCallback.ts

```diff
@@ -16,27 +15,34 @@ import type { EC3Token } from "./EC3Token";
 *   });
 * } else {
 */
-export function handleEC3AuthCallback(props: EC3ConfigProps) {
-  const MILI_SECONDS = 1000;
-  const EXPIRATRION_REDUCTION = 15 * 60;
+
+export interface EC3AuthCallbackConfigProps {
+  clientId: EC3ConfigPropsWithRedirectUri["clientId"];
+  redirectUri: EC3ConfigPropsWithRedirectUri["redirectUri"];
+  ec3Uri?: EC3ConfigPropsWithRedirectUri["ec3Uri"];
+}
+
+export function handleEC3AuthCallback(ec3Config: EC3AuthCallbackConfigProps, source: string = "ec3-auth") {
+  const MILLI_SECONDS = 1000;
+  const EXPIRATION_REDUCTION = 15 * 60;
```

Comment:
What does the 15 and 60 stand for? You created a constant for converted milliseconds to seconds. I think this requires more explanation than the millisecond conversion.

Reply:
To be honest, I'm not sure. I just left it as is. I changed it now to be 5 minutes. I also the constant to make it clearer.